examples/weaviate/personalized_description_with_weaviate_and_gemini_api.ipynb

```diff
@@ -0,0 +1,1119 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    PROJECT_ID = userdata.get("PROJECT_ID")
</pre><p>I set this to the empty string and it ran great for me. Do users need it? If so, what should it be set to?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>It looks like <code>project_id</code> is a required parameter in the vectorizer and generative model, however, it can be an empty string. I'll hardcode the <code>project_id</code> like this:</p>

<pre class="ql-syntax" spellcheck="false">PROJECT_ID = "" # leave this empty
API_ENDPOINT = "generativelanguage.googleapis.com"
embedding_model = "embedding-001" # embedding model 
generative_model = "gemini-pro" # language model
</pre><p> </p><p>I already flagged this to the team and hopefully it will be fixed soon!</p>